packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -1540,18 +1542,44 @@ test.serial('updateGranuleStatusToQueued() updates granule status in the databas
     { granule_id: granuleId, collection_cumulus_id: collectionCumulusId }
   );
 
-  await updateGranuleStatusToQueued({ granule: dynamoRecord, knex });
+  await updateGranuleStatusToQueued({
+    granule: dynamoRecord,
+    knex,
+  });
+
   const updatedDynamoRecord = await granuleModel.get({ granuleId });
   const updatedPostgresRecord = await granulePgModel.get(
     knex,
     { granule_id: granuleId, collection_cumulus_id: collectionCumulusId }
   );
-  const omitList = ['execution', 'status', 'updatedAt', 'updated_at'];
-  t.falsy(updatedDynamoRecord.execution);
+  const omitList = ['_id', 'execution', 'status', 'updatedAt', 'updated_at', 'files'];
+  const sortByKeys = ['bucket', 'key'];
+  const esRecord = await esGranulesClient.get(granuleId, granule.collectionId);
+  const translatedPgGranule = await translatePostgresGranuleToApiGranule({
+    granulePgRecord: updatedPostgresRecord,
+    knexOrTransaction: knex,
+  });
+
   t.is(updatedDynamoRecord.status, 'queued');
   t.is(updatedPostgresRecord.status, 'queued');
+  t.is(esRecord.status, 'queued');
+  t.is(updatedDynamoRecord.execution, undefined);
   t.deepEqual(omit(dynamoRecord, omitList), omit(updatedDynamoRecord, omitList));
   t.deepEqual(omit(postgresRecord, omitList), omit(updatedPostgresRecord, omitList));
+  t.deepEqual(sortBy(translatedPgGranule.files, sortByKeys), sortBy(esRecord.files, sortByKeys));
```

Comment:
Can we add a `t.deepEqual(omit(esRecord, omitList), omit(updatedEsRecord, omitList))` for completeness?

Reply:
[`5af1458` (#2593)](https://github.com/nasa/cumulus/pull/2593/commits/5af1458f69d17186bca5a82f4b96c811b44c2e8b)

Reply:
https://github.com/nasa/cumulus/pull/2593/files#diff-e4b95ee0e3b92433a8e670895c1e8da32d083c3ca64e31761d4bb353cbb791a7R1571